geonode/base/api/tests.py

```diff
@@ -2658,6 +2661,123 @@ def test_only_get_method_is_available(self):
         response = self.client.put(url)
         self.assertEqual(response.status_code, 403)
 
+    def test_insert_one_linked_resource(self):
+        url = reverse("base-resources-linked_resources", args=[self.doc.id])
+
+        self.client.force_login(get_user_model().objects.get(username="admin"))
+
+        response = self.client.post(url, data={"target": self.map.id})
+
+        link_connected = LinkedResource.objects.get(source_id=self.doc.id)
+
+        self.assertEqual(response.status_code, 201)
+
+        self.assertEqual(self.doc.id, link_connected.source_id)
+
+        self.assertEqual(self.map.id, link_connected.target_id)
```

Comment:
Can you also add that the `response.json()` contains the connection between the resources?
So we can be sure that the payload return the expected information